sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/OptionsBean.java

```diff
@@ -54,8 +55,11 @@ public class OptionsBean extends CourseDependentBean implements Serializable {
 	private boolean openSwitch;
 	private Calendar openDate;
 	private boolean errorflag;
-	
+	private boolean restrictExternallyManagedSections;
```

Comment:
You may have been able to achieve the same result by simply setting
managementToggleEnabled to false
as the vm templates already use this to disable or show buttons i.e.
rendered="#{optionsBean.managementToggleEnabled}"

but if that doesn't work then +1 on this

Reply:
It's not exactly the same behavior because the radio buttons are hidden instead of disabled, I don't like it, but I made the change in a different commit.

![image](https://github.com/sakaiproject/sakai/assets/8653754/b47ccef4-36f9-45f1-98b7-d209f38abaa9)